src/components/Landing/Landing.js

```diff
@@ -105,8 +105,8 @@ function Landing(props) {
           <Typography variant="subheading" gutterBottom color="secondary" align="center">
             Fast, anonymous and easy instant buying.
```

Comment:
anonymous! :D implied no registration....
Anyway, we settled a group chat for this.... why didn't interacted at all there?
I didn't want bring you rush on this small assignment and wanted to see a bit of interaction on purpose.
Did you just want finish it as fast you could?